kong/pdk/client.lua

```diff
@@ -167,15 +168,15 @@ local function new(self)
 
   ---
   -- Returns the consumer from the datastore.
-  -- Will look up the consumer by id, and optionally will do a second search by name.
+  -- Looks up the consumer by ID, and can optionally do a second search by name.
   -- @function kong.client.load_consumer
   -- @phases access, header_filter, response, body_filter, log
-  -- @tparam string consumer_id The consumer id to look up.
+  -- @tparam string consumer_id The consumer ID to look up.
   -- @tparam[opt] boolean search_by_username. If truthy,
```

Comment:
Is `search_by_username.` supposed to have a `.` at the end, or is it a typo? Same goes for `allow_terminated.` in this file. There are no other functions anywhere in the PDK doc that have the `.` at the end, so I'd like to double-check.

Reply:
It looks like a typo/stylistic oversight to my eyes. I vote :+1: for removing it.